ci/ci/build.py

```diff
@@ -504,7 +504,11 @@ def __init__(self, params, namespace_name, admin_service_account, public, secret
         self.job = None
 
         if is_test_deployment:
-            self._name = DEFAULT_NAMESPACE
+            assert self.namespace_name in ('default', 'batch_pods'), self.namespace_name
```

Comment:
I'd think this would be the opposite. Where if it's a test deployment, the namespaces won't be default, batch_pods.

Reply:
In both build.yaml and ci/test/resources/build.yaml, we use namespaces named batch-pods and default.

The `namespace_name` identifies either default or batch-pods. The `_name` is the actual name of the namespace assigned to represent default or batch-pods for this particular test/ci/deploy job.

Reply:
Ok. Thanks for the explanation.